backend/Tournament/views.py

```diff
@@ -123,4 +126,50 @@ def delete(self, request, tournament_id, tournamentplayer_id):
 		if tournament_player.user == request.user:
 			return Response({'error': 'Cannot delete host tournament player'}, status=status.HTTP_404_NOT_FOUND)
 		tournament_player.delete()
-		return Response(status=status.HTTP_204_NO_CONTENT)
\ No newline at end of file
+		return Response(status=status.HTTP_204_NO_CONTENT)
+
+class TournamentMatchListView(APIView):
+	def get(self, request, tournament_id):
+		tournament = Tournament.objects.filter(id=tournament_id).first()
+		if not tournament:
+			return Response(f"Tournament {tournament_id} not found.", status=status.HTTP_404_NOT_FOUND)
+		tournament_matches = tournament.matches.all().order_by('id')
+		tournament_match_serializers = TournamentSerializers.match(tournament_matches, many=True)
+		return Response(tournament_match_serializers.data, status=status.HTTP_200_OK)
+
+class TournamentMatchDetailView(APIView):
+	def get(self, request, tournament_id, tournament_match_id):
+		'''get details of a single tournament match.
+		If host and tournament is in progress and the tournament_match_id is the next_match, return match url.
+		'''
+		tournament = Tournament.objects.get(id=tournament_id)
+		if not tournament:
+			return Response(f"Tournament {tournament_id} not found", status=status.HTTP_404_NOT_FOUND)
+		tournament_matches = tournament.matches.all().order_by('id')
+		if tournament_match_id >= tournament_matches.count():
+			return Response(f"Tournament {tournament_id} does not have {tournament_match_id} match", status=status.HTTP_404_NOT_FOUND)
+		
+		# check if user is host, tournament is in progress, and the tournament_match_id is the next_match, if so, start match
+		if request.user == tournament.host_user \
```

Comment:
A long if statement like that could be pulled out into a function with a descriptive name. Makes the comment obsolete and the code easier to read